webpack/loaders/manifest-loader.ts

```diff
@@ -1,26 +1,27 @@
-'use strict';
+/* eslint-disable no-invalid-this */
```

Comment:
Not sure should we add an override in `.eslintrc`?

Reply:
I think we should turn off no-invalid-this in typescript files and use @typescript/no-invalid-this instead which should be able to handle this code without a disable?

Reply:
I agree, should I add it in this PR or you'll do a new one?

Reply:
I think you doing it in this PR or another PR is fine so that you can verify that it removes the need for this disable.

Reply:
Done, you're right, this rule is the correct one.